$OpenBSD: patch-protocols_jabber_libiris_src_xmpp_base64_base64_cpp,v 1.1 2019/08/25 21:43:39 cwen Exp $

fix the build on archs where char is unsigned by default (ppc, arm)

Index: protocols/jabber/libiris/src/xmpp/base64/base64.cpp
--- protocols/jabber/libiris/src/xmpp/base64/base64.cpp.orig
+++ protocols/jabber/libiris/src/xmpp/base64/base64.cpp
@@ -45,7 +45,7 @@ QByteArray Base64::decode(const QString& input)
 	// 64 specifies eof
 	// everything else specifies data
 
-	char tbl[] = {
+	signed char tbl[] = {
 		-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,
 		-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,
 		-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,62,-1,-1,-1,63,
